Source/WebCore/rendering/mathml/RenderMathMLRoot.cpp

```diff
@@ -200,11 +202,14 @@ void RenderMathMLRoot::layoutBlock(bool relayoutChildren, LayoutUnit)
         return;
     }
 
+    layoutFloatingChildren();
+
     // We layout the children, determine the vertical metrics of the base and set the logical width.
     // Note: Per the MathML specification, the children of <msqrt> are wrapped in an inferred <mrow>, which is the desired base.
     LayoutUnit baseAscent, baseDescent;
     recomputeLogicalWidth();
     computeAndSetBlockDirectionMarginsOfChildren();
+
```

Comment:
nit: please remove this whitespace change

Reply:
Fixed